sdk/src/types/block/semantic/mod.rs

```diff
@@ -297,9 +297,6 @@ impl<'a> SemanticValidationContext<'a> {
                     if output.features().block_issuer().is_some() {
                         let account_id = output.account_id_non_null(&OutputId::new(self.transaction_id, index as u16));
 
-                        if self.commitment_context_input.is_none() {
```

Comment:
Are these errors staying, or getting removed in future?

Reply:
You mean the variants? They are still being used on the input side. Only output side can be checked syntactically.